Resources/Prototypes/Entities/Objects/Tools/tools.yml

```diff
@@ -181,10 +181,15 @@
     qualities:
       - Pulsing
   - type: SignalLinker
+  - type: NetworkConfigurator
   - type: UserInterface
     interfaces:
     - key: enum.SignalLinkerUiKey.Key
       type: SignalPortSelectorBoundUserInterface
+    - key: enum.NetworkConfiguratorUiKey.List
+      type: NetworkConfiguratorBoundUserInterface
+    - key: enum.NetworkConfiguratorUiKey.Configure
+      type: NetworkConfiguratorBoundUserInterface
```

Comment:
im kind of iffy of just everything going on the multitool, but this doesnt conflict with anything else right now, so I guess its alright